### PR TITLE
user stats

### PR DESCRIPTION
**Pull request by chrisweber1408**
user can see his game stats in a doughnut chart

### CODE REVIEW

File: frontend/src/Pages/InfoPage.tsx [hunk 1]
```diff
@@ -129,6 +130,14 @@ export default function InfoPage() {
...
             </form>
+            <Grid container>
+                <Grid item xs={6} textAlign={"center"}>
+                    <SpentMoneyDoughnut Game={game}/>
+                </Grid>
+                <Grid item xs={6} textAlign={"center"}>
+                    <MoneyVsPlaytimeDoughnut Game={game}/>
+                </Grid>
+            </Grid>
         </div>
```
Comment (NicolaiKopka) [inline frontend/src/Pages/InfoPage.tsx:141]: kommt <Grid> von Material UI ?

File: frontend/src/components/charts/MoneyVsPlaytimeDoughnut.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import {UserGameDTO} from "../../service/model";
+import {Doughnut} from "react-chartjs-2"
+import {Chart, ArcElement, Legend, Tooltip, Title} from "chart.js"
+Chart.register(ArcElement, Legend, Tooltip, Title);
+
+interface gameDoughnutProps{
+    Game: UserGameDTO
```
Comment (NicolaiKopka) [inline frontend/src/components/charts/MoneyVsPlaytimeDoughnut.tsx:7]: would probably make variable names lowerCase. Shouldn't be a problem here but always seems like a class name

**Review by NicolaiKopka — APPROVED** [review]

File: frontend/src/Pages/InfoPage.tsx (hunk 1 above)
Comment (chrisweber1408) [reply frontend/src/Pages/InfoPage.tsx:141]: Ja genau das <Grid> kommt von Material UI, komm damit auf jeden fall sehr gut klar 😄 👍

File: frontend/src/components/charts/MoneyVsPlaytimeDoughnut.tsx (hunk 2 above)
Comment (chrisweber1408) [reply frontend/src/components/charts/MoneyVsPlaytimeDoughnut.tsx:7]: oh yes, didn't know why i write it in upperCase here, i will change it 👍

**Pull request merged**